ibm_catalog.json

```diff
@@ -609,7 +609,7 @@
         "solution"
       ],
       "short_description": "Deploys a secure VPC network without compute resources",
-      "long_description": "The VPC landing zone deployable architecture deploys a simple Virtual Private Cloud (VPC) infrastructure without any compute resources, such as Virtual Server Instances (VSI) or Red Hat OpenShift clusters. You can use this architecture as a base on which to deploy compute resources. Or you can deploy those resources by using the other landing zone deployable architectures: VSI on VPC landing zone and Red Hat OpenShift Container Platform on VPC landing zone.",
+      "long_description": "The VPC landing zone deployable architecture deploys a simple Virtual Private Cloud (VPC) infrastructure without any compute resources, such as Virtual Server Instances (VSI) or Red Hat OpenShift clusters. You can use this architecture as a base on which to deploy compute resources. Or you can deploy those resources by using the other landing zone deployable architectures: [VSI on VPC landing zone](https://cloud.ibm.com/catalog/architecture/deploy-arch-ibm-slz-vsi-ef663980-4c71-4fac-af4f-4a510a9bcf68-global) and [Red Hat OpenShift Container Platform on VPC landing zone](https://cloud.ibm.com/catalog/architecture/deploy-arch-ibm-slz-ocp-95fccffc-ae3b-42df-b6d9-80be5914d852-global?catalog_query=aHR0cHM6Ly9jbG91ZC5pYm0uY29tL2NhdGFsb2c%2Fc2VhcmNoPVJlZCUyNTIwSGF0JTI1MjBPcGVuU2hpZnQlMjUyMENvbnRhaW5lciUyNTIwUGxhdGZvcm0lMjUyMG9uJTI1MjBWUEMlMjUyMGxhbmRpbmclMjUyMHpvbmUjc2VhcmNoX3Jlc3VsdHM%3D).",
```

Comment:
you can remove this part `?catalog_query=aHR0cHM6......`

Reply:
Done the changes.Re-triggerring the pipeline.